phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less

```diff
@@ -304,14 +304,11 @@
 /*******************************************************/
 .leftPane {
     -fx-background-color: @main-color;
-    -fx-min-width: 15em;
-    -fx-pref-width: 15em;
     -fx-padding: 0 0 0.83em 0;
 
     .leftPaneScrollbar {
         -fx-border-width: 0;
```

Comment:
Yes. But I don't want to change this now. As I said above, it's a general task to cleanup the CSS which does not belong to this PR.